Recommend removal of sensitive HTTP headers (#20774)
DataDog/documentation: content/en/real_user_monitoring/guide/proxy-rum-data.md

3. Forward the request to the Datadog intake URL using the POST method.
4. Leave the request body unchanged.

> <div class="alert alert-info">For security reasons, remove any HTTP headers potentially containing sensitive information, e.g. the `cookie` header.</div>

Reviewer: ```suggestion
<div class="alert alert-warning">For security reasons, remove any HTTP headers that potentially contain sensitive information, such as the <code>cookie</code> header.</div>
```
